pkgs/applications/networking/syncthing/default.nix

```diff
@@ -5,6 +5,7 @@ let
     buildGoModule rec {
       version = "1.4.0";
       name = "${stname}-${version}";
```

Comment:
is name still needed ?

Reply:
Name or pname, but I don't want to modify the package any more than necesarry.

Reply:
You might as well split it into pname and version while at it.